components/js_lang/js_lang_common.h

```diff
@@ -28,13 +28,15 @@
 /*
   Services and helper headers provided by SQL core which our component uses.
 */
+#include <mysql/components/services/bits/mysql_thd_attributes_bits.h>
```

Comment:
Is this include needed?

Reply:
This has been removed in the second patch.